### DISCUSSION TITLE
Help: 閲覧権限のないツリーのツリー名を隠したい

### DISCUSSION THREAD

**Discussion by rtakamura** [growilabs/growi#9409] [Help / Questions] [answered]
閲覧権限のないユーザーがアクセスした場合も、全てのツリーが表示され、クリックすると「Forbidden」が表示されます。
これをツリー名ごと表示しないという設定があるとよりセキュアに利用できると思います。
理想は、親ツリーの閲覧権限が無ければ、サブツリーのタイトルも開けないという感じです。
企業で利用していますが、プロジェクト毎のツリーがあり、参加していないツリーはクリックしても「Forbidden」で中身は見れませんが、
全てのツリー名が全て見えるとある程度で判断できてしまうのと、開けないツリーが大量に見えるより、そのユーザーが必要なツリーが出ている方が利便性が良いと思いました。
サービスを複数立ち上げるという手もありますが、そうなるとプロジェクト毎にサービスを立ち上げる必要があり、サーバー管理側の管理負担や社員のアクセス先をいくつも管理する形になるので、
閲覧権限のないページはツリー名を見せないという手法で解決できれば、より情報ポータルとして利用しやすいと思います。
ご検討いただけますと幸いです。

**Comment by yuki-takei** [comment, marked as answer]
Security Settings の設定を変更してみてください

https://docs.growi.org/ja/admin-guide/management-cookbook/security.html#%E3%82%BB%E3%82%AD%E3%83%A5%E3%83%AA%E3%83%86%E3%82%A3%E8%A8%AD%E5%AE%9A-2

過去のバージョンではスイッチの ON/OFF と効果が逆転するバグを含んでいたものもありますので、切り替えて挙動を確認してみてください。

**Comment by rtakamura** [comment]
ご回答ありがとうございます。
ご教授いただきました設定項目について、 ON/OFFを逆にしておりました。
スイッチを切り替え、リストに表示されなくなったことを確認いたしました。
